src/components/manage/s3/LrS3Account.vue

```diff
@@ -0,0 +1,323 @@
+                    ref="password"
+                    v-model="passwordForm.password"
+                    outlined
+                    :rules="passwordRules"
+                    validate-on-blur
+                  ></v-text-field>
+                </v-col>
+                <v-col cols="12" sm="6">
+                  <label for="confirmPassword">Confirm Password *</label>
+                  <v-text-field
+                    name="confirmPassword"
+                    ref="confirmPassword"
+                    v-model="passwordForm.confirmPassword"
+                    :rules="confirmPasswordRules"
+                    outlined
+                    validate-on-blur
+                  ></v-text-field>
+                </v-col>
+              </v-row>
+            </v-container>
+          </div>
+        </v-card-text>
+        <v-card-actions>
+          <v-btn color="csmprimary" @click="passwordDialog = false" outlined
+            >cancel</v-btn
+          >
+          <v-btn color="csmprimary" @click="resetPassword()" dark>Reset</v-btn>
+        </v-card-actions>
+      </v-card>
+    </v-dialog>
+
+    <v-dialog v-model="generatedAccessKeyDialog" max-width="800px" persistent>
+      <v-card>
+        <v-card-title>
+          <div class="title-container">
+            <LrSvgIcon icon="green-tick.svg" class="title-icon" />
+            <div class="title-content">Access key created</div>
+          </div>
+        </v-card-title>
+        <v-divider></v-divider>
+        <v-card-text>
+          <div class="content-container">
+            <v-container>
+              <v-row class="content-title">
+                <v-col>
+                  <v-icon class="content-title-icon" color="csmwarning"
+                    >mdi-alert-circle-outline</v-icon
+                  >Save this information, you will not see it again. Download as
+                  CSV and close.
+                </v-col>
+              </v-row>
+              <v-row>
+                <v-col cols="3">
+                  <b>Access Key :</b>
+                </v-col>
+                <v-col>{{ newAccessKey.access_key_id }}</v-col>
+              </v-row>
+              <v-row>
+                <v-col cols="3">
+                  <b>Secret Key :</b>
+                </v-col>
+                <v-col>{{ newAccessKey.secret_key }}</v-col>
+              </v-row>
+              <v-row>
+                <v-col cols="3">
+                  <b>Note :</b>
+                </v-col>
+                <v-col
+                  >This deployment is not configured with Virtual IP. Please
+                  check documentation to get IP Address of public data
+                  interface</v-col
+                >
+              </v-row>
+            </v-container>
+          </div>
+        </v-card-text>
+        <v-divider></v-divider>
+        <v-card-actions class="pa-4">
+          <v-btn color="csmprimary" @click="downloadKey()" dark
+            >download & close</v-btn
```

Comment:
There was a discussion on this text. Should it be "Download and Close" or "Save and Close"

Reply:
Download and close is final I think.